main.js

```diff
@@ -0,0 +1,11 @@
+const api = new APIManager()
+
+
+const data = function() {
+    api.loadData()
+}
+
+const displayData = () => { 
+    const render = new Renderer(api.data)
```

Comment:
See my comment about the constructor in the `Renderer.js`. If you change it then you can instantiate this once at the top of the file.